### PR TITLE
Add the visual basic and msbuild language

### PR DESCRIPTION
**Pull request by TyOverby**
I work on [Roslyn](https://github.com/dotnet/roslyn) and wanted to see how our VB usage compares to c# internally.  With this PR I get the following which seems about right.

```
-------------------------------------------------------------------------------
 Language            Files        Lines         Code     Comments       Blanks
-------------------------------------------------------------------------------
 Batch                  24          270          193           14           63
 C Header                1           15           10            1            4
 C#                   7956      2559827      2023968       218514       317345
 C++                     4          185          135           22           28
 F#                      2           96           63           10           23
 JSON                   61          869          869            0            0
 MSBuild               285        20983        19774          980          229
 Markdown              118        11687        11687            0            0
 Shell                  14          474          305           73           96
 Plain Text            157       480277       480277            0            0
 Visual Basic         3226      1809279      1424731       132636       251912
 XAML                   26         2957         2849           18           90
 XML                    59        20389        15997         1986         2406
-------------------------------------------------------------------------------
 Total               11933      4907308      3980858       354254       572196
-------------------------------------------------------------------------------
```

### CODE REVIEW

**Comment by Aaronepower** [comment]
Thank you for your PR and congrats on your first contribution! If you could change `Vb` to `VisualBasic` and `MSBuild` to `MsBuild` in `languages.json` so that the generated code is in the correct casing, I would be happy to merge it in.

**Comment by TyOverby** [comment]
@Aaronepower: thanks for the feedback!  I've updated as suggested.

**Comment by Aaronepower** [comment]
@TyOverby Sorry this has taken long to merge in due to appveyor failing. Can you resolve the new merge conflict and I'll merge it in regardless of appveyor.

**Comment by TyOverby** [comment]
@Aaronepower: done.

File: languages.json [hunk 1]
```diff
@@ -1265,6 +1276,16 @@
...
+        "MsBuild":{
+            "name":"MSBuild",
+            "base":"html",
+            "extensions":[
+                "csproj",
+                "vbproj",
+                "fsproj",
+                "props",
+                "targets"
+
```
Comment (Aaronepower) [inline languages.json]: @TyOverby Missing a closing `]` and `},`

**Comment by TyOverby** [comment]
Ah yes, sorry, I made my fix in the github editor and it was a bit overzealous with deletes :+1:

**Comment by Aaronepower** [comment]
Thanks for the PR!

**Pull request merged**